reactivesocket-cpp/src/Frame.cpp

```diff
@@ -30,6 +29,10 @@ std::unique_ptr<folly::IOBuf> FrameBufferAllocator::allocateBuffer(
   return folly::IOBuf::createCombined(size);
 }
 
+std::unique_ptr<folly::IOBufQueue> FrameBufferAllocator::allocateQueue() {
```

Comment:
what is the value of this function?
In the case of allocateBuffer, we can actually override the behavior and allocate extra bytes (for message frame size). In the case of allocateQueue, do we plan on doing anything extra? If the answer is no, then I would suggest removing this method and allocate IOBufQueue on stack in the serializeOut methods